aiven/client/speller.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021, Aiven, https://aiven.io/
+#
+# This file is under the Apache License, Version 2.0.
+# See the file `LICENSE` for details.
+
+# Spelling Corrector in Python 3; see http://norvig.com/spell-correct.html
+#
+# Copyright (c) 2007-2016 Peter Norvig
+# MIT license: www.opensource.org/licenses/mit-license.php
+
+
+def suggest(word_to_check, word_dict):
+    def get_correction(word):
```

Comment:
would it make sense to extract these small helper functions out of `suggest`, so that it will be easier to observe what does `suggest` exactly do?

Reply:
I don't agree with this. I think that this function is quite small and is good as is.